package.json

```diff
@@ -24,42 +23,16 @@
         "stream-events": "https://registry.npmjs.org/stream-events/-/stream-events-1.0.2.tgz",
         "string-format-obj": "https://registry.npmjs.org/string-format-obj/-/string-format-obj-1.1.0.tgz",
         "through2": "https://registry.npmjs.org/through2/-/through2-2.0.3.tgz"
-      }
+      },
+      "version": "https://registry.npmjs.org/@google-cloud/common/-/common-0.13.6.tgz"
     },
     "@google-cloud/functions-emulator": {
-      "version": "https://registry.npmjs.org/@google-cloud/functions-emulator/-/functions-emulator-1.0.0-alpha.25.tgz",
```

Comment:
What's the reason for all the changes here?
If you revert this file (undo all changes), will your cloud function work?